### PR TITLE
Task 1 selenium wd

### PR DESCRIPTION
**Pull request by OleksandrBerezhnyi**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -30,5 +30,12 @@
             <version>3.141.59</version>
         </dependency>
 
+        <dependency>
+            <groupId>io.github.bonigarcia</groupId>
+            <artifactId>webdrivermanager</artifactId>
+            <version>4.2.2</version>
```
Comment (DmitriyBondarenko) [inline pom.xml]: Please, create separate `<properties> </properties>` section in your pom.xml and move all dependencies versions to this section.

File: src/test/java/ua/yakaboo/BaseTests.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class BaseTests extends WebDriverSettings {
```
Comment (DmitriyBondarenko) [inline src/test/java/ua/yakaboo/BaseTests.java]: Please, rename your `WebDriverSettings ` class to `BaseTest`, and `BaseTests` to smth like `CommonTests`

File: src/test/java/ua/yakaboo/BaseTests.java [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+        String secondTab = "Книжковий блог Yakaboo — Головна сторінка";
+        driver.findElement(By.xpath(".//ul[@class='cms-list']//a[contains(@href,'blog')]")).click();
+        switchWindowByTitle(secondTab, driver);
+        Thread.sleep(2000);
+        WebElement blogPage = driver.findElement(By.xpath(".//div[@class='elementor-image']//a[contains(@href,'blog')]"));
+        Assert.assertTrue(blogPage.isDisplayed(), "Blog page does not open");
+        switchWindowByTitle("Yakaboo — Інтернет-магазин книг, подарунків і дитячих товарів. Купити книги та подарунки по найкращим в Україні цінами.", driver);
+    }
+
+    public void switchWindowByTitle(String title, WebDriver driver) {
```
Comment (DmitriyBondarenko) [inline src/test/java/ua/yakaboo/BaseTests.java]: Please, move this method to some helper class.

File: src/test/java/ua/yakaboo/WebDriverSettings.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+import org.testng.annotations.*;
+
+public class WebDriverSettings {
+
+    public ChromeDriver driver;
+
+    @BeforeMethod
+    public void setUp() {
+        System.setProperty("webdriver.chrome.driver", "src/main/resources/drivers/chromedriver87");
+        driver = new ChromeDriver();
```
Comment (DmitriyBondarenko) [inline src/test/java/ua/yakaboo/WebDriverSettings.java]: Use WebDriver manager for driver creation

**Review by DmitriyBondarenko — CHANGES_REQUESTED** [review]
Fix comments

File: pom.xml (hunk 1 above)
Comment (OleksandrBerezhnyi) [reply pom.xml]: Created

File: src/test/java/ua/yakaboo/BaseTests.java (hunk 2 above)
Comment (OleksandrBerezhnyi) [reply src/test/java/ua/yakaboo/BaseTests.java]: Renamed

File: src/test/java/ua/yakaboo/BaseTests.java (hunk 3 above)
Comment (OleksandrBerezhnyi) [reply src/test/java/ua/yakaboo/BaseTests.java]: Done

File: src/test/java/ua/yakaboo/WebDriverSettings.java (hunk 4 above)
Comment (OleksandrBerezhnyi) [reply src/test/java/ua/yakaboo/WebDriverSettings.java]: Done

**Review by DmitriyBondarenko — APPROVED** [review]

**Pull request merged**